tools/linker/CoreOptimizeGeneratedCode.cs

```diff
@@ -935,6 +946,28 @@ int ProcessIsARM64CallingConvention (MethodDefinition caller, Instruction ins)
 			return 0;
 		}
 
+		void ProcessRuntimeArch (MethodDefinition caller, Instruction ins)
+		{
+			const string operation = "inline Runtime.Arch";
+
+			if (Optimizations.InlineRuntimeArch != true)
```

Comment:
I know that this was in the previous code, but whenever a see a comparison to a constant bool I get twitchy (ask me why sometime)

Reply:
That's because `InlineRuntimeArch` is a nullable bool. In this particular case, the if condition is true if `Optimizations.InlineRuntimeArch` is either `false` or `null`. It's the short version of:

```csharp
if (!Optimizations.InlineRuntimeArch.HasValue || !Optimizations.InlineRuntimeArch.Value)
    ....
```